lib/metasploit/framework/login_scanner/base.rb

```diff
@@ -57,8 +57,8 @@ module Base
                     inclusion: { in: [true, false] }
 
           validates :bruteforce_speed,
-                    presence: false,
                     numericality: {
+                      allow_nil: true,
```

Comment:
In this case that works but in general I disagree, unless: "bruteforce_speed.nil?" would skip any other validators.